lib/expressView.js

```diff
@@ -20,7 +20,30 @@ var format = require('util').format;
 var inherit = require('util').inherits;
 var debug = require('debug')(require('../package').name);
 
-var View = util.safeRequire('express/lib/view');
+var ExpressView = util.safeRequire('express/lib/view');
```

Comment:
Here instead of just bringing in the express `View` we bring it in under a different variable name `ExpressView`. The goal is to overwrite the main `View` function but keep all of the existing `prototypes that view has` (more on this down below).